spring-thrift-starter/src/main/java/info/developerblog/spring/thrift/client/pool/ThriftClientPooledObjectFactory.java

```diff
@@ -74,7 +100,7 @@ public TServiceClient create(ThriftClientKey key) throws Exception {
     public void activateObject(ThriftClientKey key, PooledObject<TServiceClient> p) throws Exception {
         super.activateObject(key, p);
 
-        Optional requestId = Optional.fromNullable(MDC.get(requestIdLogger.getMDCKey()));
+        Optional requestId = Optional.ofNullable(MDC.get(requestIdLogger.getMDCKey()));
```

Comment:
I don't have any concrete values but I think that a lot of people still use Java 7. That's why I'm asking. Sleuth is still Java 7 compatible.